packages/encode/src/index.ts

```
@@ -0,0 +1,22 @@
+import type { Base64Encoder } from './base64/base64.types';
+
+const isNodeJs = globalThis.process !== undefined;
+const isBrowser = globalThis.window !== undefined;
+
+let b64: Base64Encoder;
+
+if (globalThis.atob !== undefined) {
+  b64 = await import('./base64/base64.browser.js').then(
```

ESLint

[error] Unable to resolve path to module './base64/base64.browser.js'.

(import-x/no-unresolved)